CSharp/BatchExplorer/Helpers/BasicDataProvider.cs

```diff
@@ -1,137 +1,151 @@
-﻿//Copyright (c) Microsoft Corporation
```

Comment:
I am fine waiting on the whole PR to hear on this specific CR if it's controversial. Rather than doing work just to do work and requiring re-testing. Pointing out that the CRLF is not that solid of a reason for seeing the code changes themselves.